infrastructure/tooling/src/importer/util.js

```diff
@@ -1,35 +1,24 @@
 const _ = require('lodash');
 const azure = require('fast-azure-storage');
 const {postgresTableName} = require('taskcluster-lib-entities');
-const assert = require('assert').strict;
 
 exports.fail = msg => {
   console.error(msg);
   process.exit(1);
 };
 
-exports.requireEnv = name => {
-  if (process.env[name]) {
-    return process.env[name];
-  }
-  throw new Error(`$${name} must be given`);
-};
-
-// number of table (or table-chunk) imports to run in parallel
-exports.CONCURRENCY = 30;
-
 // tables that are allowed to migrate
 exports.ALLOWED_TABLES = [
-  // 'Clients',
-  // 'Hooks',
+  'Clients',
```

Comment:
Most imports will likely run on a dev deployment to make sure they work before running it on the actual deployment. Should we have a flag that includes/excludes signing/encrypted tables?

Reply:
sounds good